opennms-webapp/src/main/webapp/admin/userGroupView/roles/view.jsp

```diff
@@ -148,14 +149,14 @@
 
     <tr>
       <th>Description</th>
-  	<td colspan="3">${role.description}</td>
+  	<td colspan="3"><%= request.getAttribute("role") == null ? "" : WebSecurityUtils.sanitizeString(((WebRole) request.getAttribute("role")).getDescription()) %></td>
```

Comment:
Is there any reason why this one contains a null check, but the others on this page don't?